### ISSUE TITLE
Test of beta analysis interface

### ISSUE THREAD

**Issue by vsoch** [neurosynth/neurosynth-web#84]
This is so great! And I even got the user name "vanessa." Here is some initial feedback, in case it helps. You likely have thought of all this already!

1. As a user, my first instinct when I come to the new analysis page is to search for studies (and of course I didn't read the instructions). It's a little confusing when the "search" for the selected studies doesn't immediately let me do any kind of search (because I haven't selected anything). Maybe if there are no selected studies, the "All Studies" tab should be selected by default?

![interface](https://cloud.githubusercontent.com/assets/814322/7394462/8f65548a-ee48-11e4-8385-d08af4c47273.png)

2. It would be nice to be able to select studies with checkboxes versus "add all" based on the filters.

![selected](https://cloud.githubusercontent.com/assets/814322/7394477/b81aa3f8-ee48-11e4-82cc-2f1ea1cb497b.png)

3. I love the analysis interface with my saved analyses on the left!

4. Lastly, I think the "study counter" might have a bug, and there are some issues with spacing of the layer names.

![last](https://cloud.githubusercontent.com/assets/814322/7394535/166839fc-ee49-11e4-851c-ef0d03b5afa6.png)

Great job Tal! This is ridiculous. We need to catch up on level of coolness in NeuroVault :)

**Comment by asfox** [comment]
Tal & Skept,

This is amaaaaazing.

What is the most efficient way for me to report any bugs that I come
across?

For example...
I ran into a what I think is a bug on the custom analysis page. (
http://www.neurosynth.org/analyses/custom/ )
- I added more studies than I really wanted (think: pain), then I want to
remove a bunch by search (think: placebo). First, there is no button to
mass-remove. You might want to make this a thing. More importantly, when I
searched in the "Selected Studies" tab, the "remove" button for individual
studies stopped working.
- The remove button also fails to respond when I click to another set of
studies (i.e. Next page, or results 11-20)
- The remove button also fails when I add 100 studies.


Also, will there at some point be a way to upload a text file of desired
meta-analyses, or is this only for people who want to download and run the
code?

AMAAAAAAAAZING,
:-Drew

**Comment by tyarkoni** [comment]
There's a  button for "remove all filtered", but it's on the All Studies tab rather than Selected Studies tab, so that might be a little unintuitive. Thanks for flagging the other bugs. @skept, you should probably keep track of the feedback we're getting and then we can prioritize enhancements/fixes for the next release.

I hadn't thought about the possibility of uploading a file. That's easily doable if the idea is just to select those studies in Neurosynth that are in the file (e.g., by PubMed ID), but making it so you can enter new studies and coordinates is going to be a much bigger slog (definitely on the agenda, but that's a minimum of 6 months out, I'd say). Once the latter is done, accepting text files would probably be pretty straightforward.

**Comment by tyarkoni** [comment]
Oh, and thanks for the feedback as well, @vsoch! Keep it coming!

**Comment by tyarkoni** [comment]
Also, @skept, someone else just emailed me about the same issue with the removal button not working on pages past the first. Probably good to prioritize that fix when you get a chance. Will open a new issue for it.

**Comment by skept** [comment]
Thanks for the feedback!

@vsoch I like the idea of making the "All Studies" tab the default when no studies have been selected. Regarding point 1, on the "All Studies" tab you can select/deselect individual studies by clicking on their corresponding row (anywhere except the title link). I agree having a checkbox on each row would be more intuitive - although it's a little more complicated to implement :-)

@tyarkoni I'm looking at the remove bug. Yes, I'll be creating new issues for the next release based on the feedback we're getting.

**Comment by vsoch** [comment]
Ah, I see. Clicking the row would work too, but there might be an instruction somewhere that we can do that?
